Tools/CISupport/build-webkit-org/config.json

```diff
@@ -411,224 +411,252 @@
                       "triggers": ["gtk-linux-64-release-tests", "gtk-linux-64-release-tests-js", "gtk-linux-64-release-tests-webdriver",
                                    "gtk-linux-64-release-wayland-tests", "gtk-linux-64-release-perf-tests",
                                    "gtk-linux-64-release-skip-failing-tests"],
+                      "additionalArguments": ["--no-fatal-warnings"],
                       "workernames": ["gtk-linux-bot-2"]
                     },
                     {
                       "name": "GTK-Linux-64-bit-Release-Tests", "factory": "TestAllButJSCFactory",
                       "platform": "gtk", "configuration": "release", "architectures": ["x86_64"],
+                      "additionalArguments": ["--no-fatal-warnings"],
```

Comment:
Won't this break the test steps? `run-webkit-tests` complains when given this option, likely other test scripts too.

Maybe instead of changing the config file we could concentrate this on the `CompileWebKit` step definition here? https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py#L2476

Reply:
Hm, OK, but is the ews-build code used by the post-commit buildbots? That's not just for EWS alone, is it?

Reply:
Um, if that steps.py is used for both EWS and post-commit bots... any idea how to tell the difference? Goal is to have fatal warnings on all the EWS bots, but not on any of the post-commit bots.

Reply:
Oh, sorry. I linked the wrong file. For post commit it's this one: https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/build-webkit-org/steps.py#L348

Reply:
> 
> BTW it looks like the build.webkit.org unit tests are failing due to what appears to be a preexisting issue:
> 

Tested locally and this failure doesn't appear in the current main, but appears in the pull request.

The cause is in https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/build-webkit-org/factories.py#L180

`BuildAndTestAllButJSCFactory` subclasses `BuildAndTestFactory` but doesn't forwards the `triggers` parameter, which should go between `architecture` and `additionalArguments` in the parent constructor invocation.

So in practice it was incidentally passing before because `additionalArguments` was empty, which is not the case in the current patch.

Reply:
Yay, great. I'll try to implement your suggestion.